models/org_team.go

```diff
@@ -646,3 +659,46 @@ func GetTeamsWithAccessToRepo(orgID, repoID int64, mode AccessMode) ([]*Team, er
 		And("team_repo.repo_id = ?", repoID).
 		Find(&teams)
 }
+
+// ___________                    ____ ___      .__  __
+// \__    ___/___ _____    _____ |    |   \____ |__|/  |_
+//   |    |_/ __ \\__  \  /     \|    |   /    \|  \   __\
+//   |    |\  ___/ / __ \|  Y Y  \    |  /   |  \  ||  |
+//   |____| \___  >____  /__|_|  /______/|___|  /__||__|
+//              \/     \/      \/             \/
+
+// TeamUnit describes all units of a repository
+type TeamUnit struct {
+	ID     int64    `xorm:"pk autoincr"`
+	TeamID int64    `xorm:"INDEX(s)"`
+	Type   UnitType `xorm:"INDEX(s)"`
+}
+
+// Unit returns Unit
+func (t *TeamUnit) Unit() Unit {
+	return Units[t.Type]
+}
+
+func getUnitsByTeamID(e Engine, teamID int64) (units []*TeamUnit, err error) {
```

Comment:
I would like this is 
```Go
func getUnitsByTeamID(e Engine, teamID int64) (units []UnitType, err error) {
     return units, e.Table("team_unit").Cols("type").Where("team_id = ?", teamID).Find(&units)
}
```
So that we could not change many `[]UnitType` to `[]*TeamUnit`.

Reply:
I wanted to be consistent with the RepoUnit method, it returns `[]*RepoUnit`